client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you-plugin-section.tsx

```diff
@@ -100,13 +102,13 @@ export const ThankYouPluginSection = ( { plugin }: { plugin: any } ) => {
 				alt={
 					translate( "%(plugin)s's icon", {
 						args: {
-							plugin: plugin.name,
+							plugin: pluginName,
```

Comment:
While this is resolving the issue, this is not enough and we should try to find the root cause of the problem. We are already decoding the name and slug fields in `normalizePluginData()`. However, I might be wrong with my thinking process. 
https://github.com/Automattic/wp-calypso/blob/0ab30f791331895435ee69707964511d76b5135c/client/lib/plugins/utils.js#L198-L203

Reply:
Thanks for your comments. I found already the root cause of the problem and that was already discussed in [this other PR](https://github.com/Automattic/wp-calypso/pull/72424#issuecomment-1400515675), but we could re-assess and iterate if we think the other approach is better. 

My main concern is iterating over a collection of plugins in a collection of sites in [here](https://github.com/Automattic/wp-calypso/blob/9701076387f9185c8dce2d6de6d62796fc539b8e/client/state/plugins/installed/actions.js#L626). But I could add some traces and do a quick performance check and then we can decide. 

Do you think we should iterate over that approach?